CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
+### 0.8.3
+
+- Added `whoami` command to see API token details.
+- Added `link` command to set up upload for an existing Pack ID instead of creating a new one.
```

Comment:
Nit, I would do `coda whoami` and `coda link` for clarity--there's a lot of functionality in the SDK beyond just the CLI tool so it will be helpful to be specific.

Reply:
Done